tutorials/connect-jamf-pro-to-smallstep.mdx

```diff
@@ -210,18 +219,57 @@ In this step, we’ll tie everything together by creating a managed policy to en
             2. Set the desired scope. This should mirror the scope you chose when creating the Policy in step 5.
             3. Choose Save
 
-Your Smallstep team is now linked to Jamf. Smallstep will do a partial sync of your device inventory from Jamf every hour, and a full sync every 8 hours.
-
 The devices that you scoped will receive a certificate and have the agent installed and running.
 
-### Confirmation
+### Option 2: Install via a software management tool
+
+If you're using a software management tool:
+
+1. Download the latest package from [packages.smallstep.com](https://packages.smallstep.com/stable/darwin/step-agent-plugin_latest.pkg)
+2. Distribute and install the package on your desired endpoints
+3. Create a user launch agent file on the endpoint, in `/Users/<USER>/Library/LaunchAgents/com.smallstep.launchd.Agent.plist` for the primary user of the device.
+
+   The Smallstep agent does not yet support multi-user deployments on macOS&mdash;it must be installed for a single user:
```

Comment:
Technically they could repeat this process for any/all users on the device, but I actually think this is a totally fine statement to make right now.